### DISCUSSION TITLE
No Community Call?

### DISCUSSION THREAD

**Discussion by MEK3DK** [microsoft/microsoft-ui-xaml#8104] [General]
I know they said they would take a break from community call, but when is it starting up again?
I was expecting there would be one this month.

**Comment by HolgerStrauss** [comment]
According to the November 2022 call video (at around 1:45), the next call should be on January 18th @ 17:00 UTC (9:00am PT).
(However, the year is not mentioned... :-))

**Reply by MEK3DK** [reply]
*Sigh* Wonder if WinUI development has just stopped?

**Reply by asierpn** [reply]
Microsoft's silence and the lack of activity in the WinUI, WinAppSDK and CsWinRT GitHub projects is very worrying.

**Reply by pjmlp** [reply]
You missed C++/WinRT as well, where after 7 years, we still don't have tooling comparable to C++/CX.

**Reply by MarkBrainfire** [reply]
WinUI 3 is supposed to be the future of Windows Desktop development.  Uno Platform empowers WinUI 3 apps to run EVERYWHERE.  In my view this new approach to developing universal native offline applications is one of the most important development stories unfolding today.  And yet we hear so little about it.  I worry about a world where every application depends upon the Internet, Azure or a store.  My worries aren't unfounded.  We've seen "Cloud" services denied for political reasons (Parler denied by AWS).  We've seen stores deny publishing for political reasons (Apple store just shuts down your app if they don't like what you think).  We've seen online purchasing / credit card processing / banking shut down for political reasons.  The Internet and Subscription Application/SAAS models are leading us to a dark world where the few dictate acceptable speech to the many.  I love ChatGTP but until laptops can run it locally it will accelerate the move to online tyranny.   

I advocate re-emphasizing DESKTOP DEVELOPMENT.  Put your BEST PEOPLE on the WinUI 3 / Windows App SDK project and reassure us that this is an on-going development focus by KEEPING YOUR COMMITMENT TO COMMUNITY CALLS and sticking to a development schedule.

The Visual Studio development team is stellar.  I'm impressed with the steady pace of solid preview releases turning out the best IDE in the industry.  WinUI developers: BE LIKE THE VISUAL STUDIO TEAM. 

Check out my website:  https://WinUI3.org

**Reply by llothar** [reply]
> WinUI 3 is supposed to be the future of Windows Desktop development.  

Just in the sense that it is the better internet client than a web browser.
I agree with your dystopian view into the future, but you are naive if you think that Microsoft is a friend here. They are the main profiteer from an all online world (and add cash free world to your worries).

Microsoft just canceled all its Virtual Reality plans to jump completely on AI because Management got a ChatGTP shock. 
Anyone thinking this will have no effect on WinUI3 is a bit naive. Just wait if we get even slower with development or if it is stalled completely.

**Reply by gautambjain** [reply]
I have invested about 3 months in WinUI 3. If not WinUI 3, what would MAUI use on Windows? 

If I have to go for an alternative to WinUI 3, what would you suggest?

**Reply by llothar** [reply]
> If I have to go for an alternative to WinUI 3, what would you suggest?

Currently i would cry and go with WPF. 
And i think MAUI is a failure by design.  You still need to write special GUI layers for each system yourself.

**Reply by JanRajnoha** [reply]
I'm using WinUI for almost 2 years and currently no problem. Connection to POS printer, UI and everything else is okay, so probably good for use, just to be aware some parts could be unfinished ... and hope, it will be fixed

**Reply by gautambjain** [reply]
Thanks @JanRajnoha . That helps.

**Reply by yki5000** [reply]
What could you suggest for C++ with low memory consumption like WinUI3, React Native, GTKMM?
(I've had several problems with a simple combo box in WinUI3 and if I look at the problems on the Issue page, when will it stop, in 10 years?)

**Reply by pjmlp** [reply]
If you care about nice tooling, can afford their licences or plan to release under FOSS licenses, Qt or C++ Builder.

**Comment by JanRajnoha** [comment]
@gabbybilka @SavoySchuler @Scottj1s Could we get more info about this situation? Why has not been call, did layoff affect WinUI projects and teams, has development stopped or paused, ... ? Since New year I didn't see bigger info from you, so I'm little worried :/

**Reply by MEK3DK** [reply]
Since the is absolute silence as @asierpn pointed out above, I think there is some reorg going on. I think the big problem is that WinUI is part of MAUI. Closing down WinUI would then hurt MAUI.
But with the current state of WinUI they probably have the option of adding significant resources or closing it down. Thus the reorg could also be that other teams are closing down and being moved to WinUI development.

**Reply by baaaaif** [reply]
[https://github.com/microsoft/WindowsAppSDK/pulls?q=is%3Apr+is%3Aclosed](https://github.com/microsoft/WindowsAppSDK/pulls?q=is%3Apr+is%3Aclosed)

It can't be completely dead, there seems to be a living 1.3, but I agree with you - that's a bit strange at the moment.
Maybe  @DrusTheAxe can comment on that

**Reply by llothar** [reply]
I doubt that anyone working in the team is allowed to comment on this. You could see in previous answers how they are dodging questions about team size, internal structure and use etc. 

>  But with the current state of WinUI they probably have the option of adding significant resources or closing it down.  

Yes, coming back from holiday and opening the github issues list was shocking. Problems like #8088 just show that the basics are still not stable. But Microsoft made clear today that they can't multifocus. It's only Azure and ChatGTP. After killing virtual reality i'm sure they have no problem leaving windows desktop development in even a worse shape than it was the last 10 years.

**Reply by pjmlp** [reply]
Whenever a customer asks me of going WinUI, I tell them the amount of times we went through rewrites, how .NET Native and C++/CX got deprecated without a second thought on the lost productivity, no designer nor blend support, and then I sum up the amount of open tickets across all repos (WinUI, AppSDK, C++/WinRT, CsWinRT) and their growing trend.

Until Office and Visual Studio bet on WinUI instead of their growing love for Web widgets, I am not advising for it.

And then we have Blazor trying to fit everywhere instead of focusing on being a good Web framework, and the whole MAUI vs WinUI and which supports what, so clearly Microsoft seems somehow lost their GUI frameworks roadmap.

**Comment by gabbybilka** [comment]
Ah! I missed seeing this when it was posted. 

Wanted to post an update alongside 1.2.3, a servicing release however it kept on getting delayed. As a quick update for this thread - 1.2.3 is coming out _very_ soon, still planning a 1.3 that focuses on quality improvements and WinAppSDK ease of use, and the community call was last-minute cancelled this month as the presenters we had planned had fell through, so I wanted to post an announcement/status update instead. I see that I was much too delayed with that, and I apologize for the worry. I'll still be posting that announcement with a few more details, ~later today.

**Reply by JanRajnoha** [reply]
Oh okay, thank you for explanation, so we don't have to be worried winui has been paused or something😊?

**Reply by gabbybilka** [reply]
Nope, WinUI 3 & WinAppSDK is in active development with a 1.2.3 release coming soon, and a 1.3 major release in the works.

**Reply by MEK3DK** [reply]
@gabbybilka I understand you probably have a lot to do and a stressful job, but please try to keep up with the minimum contact the WinUI team has with the community (the community calls).
In the perfect world there should be a person on the WinUI team that was responsible for contact with the community.
I could write a long reply like others, but I think people in this thread have made some excellent points, please read them, especially the one from @MarkBrainfire.
WinUI 3 is the only real UI desktop framework that Microsoft is developing/maintaining. Ignoring 2800+ bugs is one thing but if the connection with the community becomes "oh we just forgot" then something is really wrong, and we all start to wonder.
Thank you for your reply to this thread.

**Reply by gabbybilka** [reply]
I appreciate all of the comments from community members both in this thread and in general about how they value WinUI 3 & their expectations for the platform & team. I'm passing along the feedback and perspective to the right folks.

Regarding the January community call, that responsibility fell on my shoulders, and I definitely could have done a much better and prompter job communicating what was happening behind the scenes. Looking forward to next month's call, the discussion is posted for you to drop your questions to be answered during the live Q&A: https://github.com/microsoft/microsoft-ui-xaml/discussions/8137

**Reply by oliverw** [reply]
> Ah! I missed seeing this when it was posted.
> 
> Wanted to post an update alongside 1.2.3, a servicing release however it kept on getting delayed. As a quick update for this thread - 1.2.3 is coming out _very_ soon, still planning a 1.3 that focuses on quality improvements and WinAppSDK ease of use, and the community call was last-minute cancelled this month as the presenters we had planned had fell through, so I wanted to post an announcement/status update instead. I see that I was much too delayed with that, and I apologize for the worry. I'll still be posting that announcement with a few more details, ~later today.

2.8k open issues on this repo alone and no real progress on issues pressed by the community in two years speaks a different language though.
